rlp-derive/tests/rlp.rs

```diff
@@ -10,62 +10,62 @@ use rlp::{decode, encode};
 use rlp_derive::{RlpDecodable, RlpDecodableWrapper, RlpEncodable, RlpEncodableWrapper};
 
 #[derive(Debug, PartialEq, RlpEncodable, RlpDecodable)]
-struct Foo {
+struct Item {
 	a: String,
 }
 
 #[derive(Debug, PartialEq, RlpEncodableWrapper, RlpDecodableWrapper)]
-struct FooWrapper {
+struct ItemWrapper {
 	a: String,
 }
 
 #[test]
-fn test_encode_foo() {
-	let foo = Foo { a: "cat".into() };
+fn test_encode_item() {
+	let item = Item { a: "cat".into() };
 
 	let expected = vec![0xc4, 0x83, b'c', b'a', b't'];
-	let out = encode(&foo);
+	let out = encode(&item);
 	assert_eq!(out, expected);
 
 	let decoded = decode(&expected).expect("decode failure");
-	assert_eq!(foo, decoded);
+	assert_eq!(item, decoded);
 }
 
 #[test]
-fn test_encode_foo_wrapper() {
-	let foo = FooWrapper { a: "cat".into() };
+fn test_encode_item_wrapper() {
+	let item = ItemWrapper { a: "cat".into() };
 
 	let expected = vec![0x83, b'c', b'a', b't'];
-	let out = encode(&foo);
+	let out = encode(&item);
 	assert_eq!(out, expected);
 
 	let decoded = decode(&expected).expect("decode failure");
-	assert_eq!(foo, decoded);
+	assert_eq!(item, decoded);
 }
 
 #[test]
-fn test_encode_foo_default() {
+fn test_encode_item_default() {
 	#[derive(Debug, PartialEq, RlpEncodable, RlpDecodable)]
-	struct FooDefault {
+	struct ItemDefault {
 		a: String,
 		/// It works with other attributes.
 		#[rlp(default)]
 		b: Option<Vec<u8>>,
 	}
 
-	let attack_of = String::from("clones");
-	let foo = Foo { a: attack_of.clone() };
+	let attack_of = "clones";
+	let item = Item { a: attack_of.into() };
```

Comment:
it was a pun on `attack of clones` :(

Reply:
Clippy has exactly zero sense of humour.